submit.py

```diff
@@ -81,7 +92,7 @@
     4: 'Waiting for run',
     _RUNNING_STATUS: 'Running',
     6: 'Judge Error',
-    7: 'Submission Error',
+    # 7: '<invalid value>',
```

Comment:
Why was this removed?

Reply:
Submission Error is an old judgment status that can't occur anymore (similar to ID 15). I should have mentioned this when creating the PR.